src/Plugin/WebformHandler/CivicrmWebformHandler.php

```diff
@@ -79,6 +79,7 @@ public function defaultConfiguration() {
   }
 
   public function alterElements(array &$elements, WebformInterface $webform) {
+    \Drupal::getContainer()->get('civicrm')->initialize();
```

Comment:
This is a plugin - let's inject the 'civicrm' service instead (although, definitely call the `initialize()` here where it's needed)

Reply:
Changes look awesome! I can see you removed some other direct calls to pull this service from the container.